.travis.yml

```diff
@@ -86,7 +86,15 @@ install:
 script:
   - if [ "$PRESTASHOP_TEST_TYPE" = "unit" ]; then
         bash tests/check_file_syntax.sh;
+        file_syntax_result=$?;
         bash tests-legacy/check_phpunit.sh;
+        phpunit_result=$?;
+
+        if [[ "file_syntax_result" == "0" && "phpunit_result" == "0" ]]; then
+              exit 0;
+        else
```

Comment:
Hi @matks 
Some feedbacks ... Wdyf ?
```bash
        if [ $file_syntax_result -eq 0 ] && [ $phpunit_result -eq 0 ]; then
              exit 0;
```

Reply:
Looks better indeed 😄 I'm afraid I'm a bad Bash developer

Thanks for the help 👍

Reply:
but no not at all ... you just forgot the "$" in your vars ( `file_syntax_result` and `phpunit_result`)

Reply:
😄 
I am not proud of this PR

Fixed with your suggestion 😉